$MirOS: ports/emulators/qemu/patches/patch-target-sparc_cpu_h,v 1.3 2005/11/15 12:25:10 tg Exp $
--- target-sparc/cpu.h.orig	Sun Sep  4 17:11:09 2005
+++ target-sparc/cpu.h	Tue Nov 15 11:37:45 2005
@@ -284,8 +284,12 @@ void cpu_set_cwp(CPUSPARCState *env1, in
     } while (0)
 #endif
 
+#ifdef _BSD
+int cpu_sparc_signal_handler(int hostsignum, qemu_siginfo *info, void *puc);
+#else
 struct siginfo;
 int cpu_sparc_signal_handler(int hostsignum, struct siginfo *info, void *puc);
+#endif
 
 #include "cpu-all.h"
 
